example/examples/Downloads.tsx

```diff
@@ -21,6 +21,8 @@ const HTML = `
   </head>
   <body>
     <a href="https://www.7-zip.org/a/7za920.zip">Example zip file download</a>
+    <br>
+    <a href="http://test.greenbytes.de/tech/tc2231/attwithisofn2231iso.asis">Download file with non-ascii filename</a>
```

Comment:
Would be helpful to say in this text (or at the very least a code comment) what the expected filename is